src/cript/nodes/subobjects/condition.py

```diff
@@ -285,7 +285,7 @@ def descriptor(self, new_descriptor: str) -> None:
 
     @property
     @beartype
-    def value(self) -> Union[Number, None]:
+    def value(self) -> Optional[Union[Number, str]]:
```

Comment:
I would use `Union[Number, str, None` here just because it would easier to read in my opinion. When things start to get nested, then I start to have a hard time following it